### ISSUE TITLE
Error: base.lua

### ISSUE THREAD

**Issue by WhiteAKi** [Courseplay/courseplay#3432]
Hey,
I think the pic shows alle my problems. The game crashed after a while. Its the latest version from 15/03/19
Any Idea?

I dont know when it happens, it comes really quick. 
After I reload the game I start all machines (2 ingame helper; 2 course drescher & 3 trak on the same field; 3 other driven arround the map) and then the cam dosent follow the vehicle and i cannt do anything

[log.txt](https://github.com/Courseplay/courseplay/files/2979888/log.txt)

![WhatsApp Image 2019-03-18 at 19 53 59](https://user-images.githubusercontent.com/48693316/54555786-af46b400-49b7-11e9-9abb-ed65e5ac454d.jpeg)

**Comment by WhiteAKi** [comment]
I dont know when it happens, it comes really quick. 
After I reload the game I start all machines (2 ingame helper; 2 course drescher & 3 trak on the same field; 3 other driven arround the map)

**Comment by WhiteAKi** [comment]
[log.txt](https://github.com/Courseplay/courseplay/files/2979886/log.txt)
.

**Comment by pvaiko** [comment]
If I count these correctly these are 10 vehicles. You have to try to narrow this down to the one causing the problem otherwise we can't do much about it.

**Comment by WhiteAKi** [comment]
Oh yeah sorry, I have forgotten to write it after I post this.
After this, I checked a few things and almost everything of my intentions crashed the game.
A few days ago the _Puma 240x_ with equipment bugged into the river. Can't reset it because coursplay was still on. 
SO I saved the game and reenter it. The puma was reset as a new vehicle.
Every time I **started** coursplay on it again the game crashed.
Sold it and the problem was gone for a couple of minutes. 
Now the cam freeze and if I go out over the tab key the cam maybe reset but I can play.
The Error is still the same.
Ohh and during the time of the freeze every machine work but they don't chanche the ground
